framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -74,6 +77,7 @@
 import com.sun.source.tree.Tree;
 import com.sun.source.tree.VariableTree;
 import com.sun.source.util.TreePath;
+import com.sun.tools.javac.code.Symbol.ClassSymbol;
```

Comment:
You can probably ask for the Element corresponding to the ClassTree and then use one of the helpers in ElementUtils.